src/main/scala/org/encryfoundation/generator/Account.scala

```diff
@@ -0,0 +1,26 @@
+package org.encryfoundation.generator
+
+import java.net.InetSocketAddress
+import org.encryfoundation.common.Algos
+import org.encryfoundation.common.crypto.PrivateKey25519
+import scorex.crypto.hash.Blake2b256
+import scorex.crypto.signatures.{Curve25519, PrivateKey, PublicKey}
+import scala.io.Source
+
+case class Account(secret: PrivateKey25519, sourceNode: InetSocketAddress)
+
+object Account {
+
+  def parseFromFile(sourcePath: String): Seq[Account] = Source
+    .fromInputStream(getClass.getResourceAsStream(sourcePath))
```

Comment:
You should close BufferedSource when you finished reading data from it

Reply:
@GRYE , `.fromInputStream` automatically closes `BufferedSource ` after reading from it.